examples/src/main/java/com/mapbox/navigation/examples/core/FeedbackButtonActivity.kt

```diff
@@ -45,17 +52,20 @@ import com.mapbox.navigation.ui.feedback.FeedbackBottomSheetListener
 import com.mapbox.navigation.ui.feedback.FeedbackItem
 import com.mapbox.navigation.ui.internal.utils.ViewUtils
 import com.mapbox.navigation.ui.map.NavigationMapboxMap
-import java.lang.ref.WeakReference
 import kotlinx.android.synthetic.main.activity_feedback_button.*
+import java.lang.ref.WeakReference
 
 /**
  * This activity shows how to integrate the [com.mapbox.navigation.ui.FeedbackButton]'s
  * feedback report flow with the Navigation SDK. Road closures and
  * traffic incidents are just two of the several types of live navigation
  * feedback that can be reported.
  */
+@SuppressLint("MissingPermission")
 class FeedbackButtonActivity : AppCompatActivity(), OnMapReadyCallback,
-    FeedbackBottomSheetListener {
+        FeedbackBottomSheetListener, ArrivalObserver {
+
+    private val iconImageFeedback = "marker-feedback"
```

Comment:
At least refactor to `FEEDBACK_PIN_ICON_IMAGE_ID`?

```suggestion
    private val FEEDBACK_PIN_ICON_IMAGE_ID = "marker-feedback"
```

Moving to a companion object is the extreme option 😄 

```kotlin

    companion object {
        private const val FEEDBACK_ICON_IMAGE_ID = "marker-feedback"
    }
```

Reply:
well, I have to admit I was lazy here 🤣

Reply:
no worries. It's not like I'm using companion objects all the time in my examples either.